src/hooks/v1/transactor/RedeemTokensTx.ts

```diff
@@ -40,7 +41,10 @@ export function useRedeemTokensTx(): TransactorInstance<{
         userAddress,
         preferConverted,
       ],
-      txOpts,
+      {
+        ...txOpts,
+        title: tokenSymbol ? t`Redeem $${tokenSymbol}` : t`Redeem tokens`,
```

Comment:
I vaguely remember a util function that can turn token symbols into plural, but I might have misrememebred something else

Reply:
found it, ty!